### PR TITLE
Improve TCL true color tile docs

### PR DESCRIPTION
**Pull request by dmannarino**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [x] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: app/routes/umd_tree_cover_loss/raster_tiles.py [hunk 1]
```diff
@@ -56,7 +56,38 @@ async def umd_tree_cover_loss_raster_tile(
...
-    UMD tree cover loss raster tile.
+    UMD Tree Cover Loss raster tile.
+
+    There are two types of TCL tiles available here: True color tiles and encoded tiles.
+    The true color tiles have been pre-rendered to appear the same way TCL appears on
+    globalforestwatch.org (i.e. the tiles show pink pixels to indicate tree cover loss).
+    The caveat with the true color tiles is that ANY tree cover loss over the history of
+    the data is shown as a pink pixel. There's no way to filter out pixels which fall in
+    or out of a range of years like there is in the map interface, because that
+    information is no longer in the data!
```
Comment (danscales) [inline app/routes/umd_tree_cover_loss/raster_tiles.py]: Lines 64-67 are a bit confusing.  I would actually say something more like:

The caveat is that tile data fetched by a single tile URL is only for a specific range of years (as specified below via query parameters).  The tile directly displays the forest loss for that range of years as pink pixels.  So, the true color tiles cannot be used with a dynamic slider to adjust the year range.  To allow for a dynamically adjustable year range, you should use the encoded tiles.

File: app/routes/umd_tree_cover_loss/raster_tiles.py [hunk 2]
```diff
@@ -56,7 +56,38 @@ async def umd_tree_cover_loss_raster_tile(
...
+    The encoded tiles, on the other hand, still contain the year information but are not
+    ready to be displayed as-is. They can/need to be interpreted and rendered according to
+    the needs of the consumer of the data.
+    Note that both types of tiles are thresholded by Tree Cover Density, which means that
```
Comment (danscales) [inline app/routes/umd_tree_cover_loss/raster_tiles.py]: Maybe add in an extra blank line after 70 (i.e. start a new paragraph).

File: app/routes/umd_tree_cover_loss/raster_tiles.py [hunk 3]
```diff
@@ -56,7 +56,38 @@ async def umd_tree_cover_loss_raster_tile(
...
+
+    True color tiles (i.e. pink pixels as they appear on the map at globalforestwatch.org) thresholded by tree cover density
+        Specify the style query parameter
+        For example: https://tiles.globalforestwatch.org/umd_tree_cover_loss/v1.10/dynamic/1/1/0.png?style=tcd_10
+
+    True color tiles filtered by start and/or end year, thresholded by tree cover density
+        Specify the tcd query parameter and optionally the start/end year
+        For example: https://tiles.globalforestwatch.org/umd_tree_cover_loss/v1.10/dynamic/1/1/0.png?tcd=10&start_year=2016
+
+    Encoded tiles, so you can filter and style on your own
```
Comment (danscales) [inline app/routes/umd_tree_cover_loss/raster_tiles.py]: Just to follow the pattern of the previous two, maybe change to:

Encoded tiles (so you can filter and style on your own), thresholded by tree cover density

File: app/routes/umd_tree_cover_loss/raster_tiles.py [hunk 4]
```diff
@@ -56,7 +56,38 @@ async def umd_tree_cover_loss_raster_tile(
...
+    specify a TCD of 75. The allowable thresholds can be seen below in the query
+    parameters.
+    So if you want
+
+    True color tiles (i.e. pink pixels as they appear on the map at globalforestwatch.org) thresholded by tree cover density
+        Specify the style query parameter
+        For example: https://tiles.globalforestwatch.org/umd_tree_cover_loss/v1.10/dynamic/1/1/0.png?style=tcd_10
+
+    True color tiles filtered by start and/or end year, thresholded by tree cover density
+        Specify the tcd query parameter and optionally the start/end year
```
Comment (danscales) [inline app/routes/umd_tree_cover_loss/raster_tiles.py]: Maybe make it clearer that you can specify both start and end year by changing to:

"optionally the start year or the end year or both"

**Review by danscales — APPROVED** [review]
Looks good!  (But I suggested some clarifications)

**Review by danscales — APPROVED** [review]

**Review by tanderegg — APPROVED** [review]
@dmannarino Looks good to me, I just tested this in both ArcGIS Online and QGIS, both worked well!

**Pull request merged**